analyzers/tests/SonarAnalyzer.UnitTest/Helpers/ParameterLoaderTest.cs

```diff
@@ -213,5 +221,24 @@ private static SonarCompilationReportingContext CreateCompilationWithOption(stri
             var compilationContext = new CompilationAnalysisContext(compilation, options, _ => { }, _ => true, default);
             return new(AnalysisScaffolding.CreateSonarAnalysisContext(), compilationContext);
         }
+        private string GenerateSonarLintWithParametrizedRule(string ruleId, string key, string value)
+        {
+            var ruleParameters = new List<SonarLintXmlRule>()
+            {
+                new SonarLintXmlRule()
+                {
+                    Key = ruleId,
+                    Parameters = new List<SonarLintXmlKeyValuePair>()
+                    {
+                        new SonarLintXmlKeyValuePair()
+                        {
+                            Key = key,
+                            Value = value
```

Comment:
What's the value in this case? 
The rule parameters? 
I checked a sample of SL XML but it's not super clear to me.